cli/image_inspect.go

```diff
@@ -25,9 +25,9 @@ func (i *ImageInspectCommand) Init(c *Cli) {
 		Use:   "inspect [OPTIONS] IMAGE",
 		Short: "Display detailed information on one image",
 		Long:  imageInspectDescription,
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return i.runInspect(args)
+			return inspect.MultiInspect(args, i.runInspect)
```

Comment:
I think we should call MultiInspect in `runInspect`, because we may have some context to deal with in the future, WDYT?

Reply:
yes, the skill of dealing with context is necessary.and i will try to solve it.thanks.